index.html

```diff
@@ -23,12 +23,12 @@ <h1>Hacker Escape Rooms</h1>
 
     <nav class="nav__large">
         <ul>
-            <a href="./index.html#on-site">
-                <li>Play online</li>
-            </a>
-            <a href="./index.html#on-site">
-                <li>Play on-site</li>
-            </a>
+            <!-- <a href="./filter.htm?online"> -->
```

Comment:
The links here make a lot of sense to me. I'm not sure why you've opted to use javascript for this navigation instead.